rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] the rental should add itself to the book and person, if you call the add_rental method to do this it will add the rental to the rental this won't work, please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to see what this should look like.